src/OpenTK/Math/Vector3.cs

```diff
@@ -1574,10 +1574,9 @@ public static Vector3 Unproject(Vector3 vector, float x, float y, float width, f
         /// <returns>The result of the calculation.</returns>
         public static Vector3 operator /(Vector3 vec, float scale)
         {
-            float mult = 1.0f / scale;
-            vec.X *= mult;
-            vec.Y *= mult;
-            vec.Z *= mult;
+            vec.X /= scale;
+            vec.Y /= scale;
+            vec.Z /= scale;
             return vec;
```

Comment:
Same 'reduced divide count' optimization - we need to bench before we can change it.